api/src/main/java/com/epam/pipeline/manager/execution/SystemParams.java

```diff
@@ -69,7 +69,8 @@
     CONTAINER_CPU_RESOURCE("container-cpu-resource", "CP_CONTAINER_CPU_RESOURCE", false, true),
     CONTAINER_MEMORY_RESOURCE_POLICY("container-memory-resource-policy",
             "CP_CONTAINER_MEMORY_RESOURCE_POLICY", false, true),
-    CP_SENSITIVE_RUN("cp-sensitive-run", "CP_SENSITIVE_RUN", false, false);
+    CP_SENSITIVE_RUN("cp-sensitive-run", "CP_SENSITIVE_RUN", false, false),
+    CP_EXEC_TIMEOUT("cp-exec-timeout", "CP_EXEC_TIMEOUT");
```

Comment:
Could you also add `CP_EXEC_TIMEOUT` to `deploy/contents/install/preferences/launch.system.parameters.json`?